addons/block_code/serialization/block_script_serialization.gd

```diff
@@ -135,6 +140,47 @@ func _get_parameter_block_definition(block_name: String, parameter_name: String)
 	return block_definition
 
 
+func _get_obj_property_block_definition(block_name: String, property_value: Variant) -> BlockDefinition:
+	var block_definition: BlockDefinition
+	var variable: VariableDefinition
+	var property_name: String
+	var is_getter = true
+
+	if block_name.begins_with("get_var_"):
+		property_name = block_name.get_slice("get_var_", 1)
+	elif block_name.begins_with("set_var_"):
+		property_name = block_name.get_slice("set_var_", 1)
+		is_getter = false
+	else:
+		return null
+
+	if is_getter:
+		# Getter block needs the property's variant type information by visiting
+		# the block_code_node's parent node because the type is not saved as a
+		# key of the resource in the scene file
+		var property_info := _get_parent_node_property_info(property_name)
+		if not property_info.has("type"):
+			return null
+
+		variable = VariableDefinition.new(property_name, property_info["type"])
+		block_definition = BlocksCatalog.get_property_getter_block_definition(variable)
+	else:
+		variable = VariableDefinition.new(property_name, typeof(property_value))
```

Comment:
Why not get the parent node property for the setter, too? Then you wouldn't need to pass the value in at all.

Reply:
Oh! yes.  Good point.